src/pydata_sphinx_theme/assets/styles/base/_color.scss

```diff
@@ -5,144 +5,36 @@
 */
 html[data-theme="light"] {
   /*****************************************************************************
-  * colors
+  * main colors
   */
 
-  // main colors
   --pst-color-primary: rgb(19, 6, 84);
   --pst-color-success: rgb(40, 167, 69);
-  --pst-color-info: rgb(0, 123, 255); /*23, 162, 184;*/
+  --pst-color-info: rgb(0, 123, 255);
   --pst-color-warning: rgb(255, 193, 7);
   --pst-color-danger: rgb(220, 53, 69);
   --pst-color-text-base: rgb(51, 51, 51);
+  --pst-color-text-muted: rgb(77, 77, 77);
   --pst-color-background: rgb(255, 255, 255);
-  --pst-color-background-up: rgb(240, 240, 240); // for 3D effects
-  --pst-color-background-up-up: rgb(255, 255, 238); // for 3D effects
-  --pst-color-deactivate: rgb(77, 77, 77);
+  --pst-color-on-background: rgb(255, 255, 255);
```

Comment:
Could we add short descriptions of what the variables mean? Or a reference to bootstrap docs that explains it? For example, it's hard to understand what is the difference between `on-background` vs. `on-surface`.